tests/storage-vm

```diff
@@ -484,6 +484,20 @@ for poolDriver in $poolDriverList; do
 
         lxc delete -f v1
 
+        echo "==> Checking start / stop operations with root size.state being unset"
+        # We should be able to start a VM with migration.stateful=true eventhough the root `size.state` config is unset
+        lxc launch "${TEST_IMG:-ubuntu-minimal-daily:22.04}" v1 --vm --config migration.stateful=true -s "${poolName}"
+        # However, we should not be able to statefully stop the VM
+        ! lxc stop v1 --stateful || false
+        # But we should be able to change the root size.state config without stopping the VM
+        lxc config device override v1 root size.state=2GiB
+        # And then we should be able to statefully stop the VM
+        lxc stop v1 --stateful
+        # And then we should be able to statefully start the VM again
+        lxc start v1
```

Comment:
@gabrielmougard can you tweak this test to wait until the VM agent has restarted to confirm it was successfully restored.

Reply:
Would it make sense to also compare the boot ID to make sure the VM was live-restored and not booted from 0?
`/proc/sys/kernel/random/boot_id` has a different UUID one each boot.

Reply:
@gabrielmougard `waitInstanceBooted()` should let you wait for the VM to be usable in the `lxc exec` sense of usable.

Reply:
Sounds good idea to me!

Reply:
fixed